src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -400,7 +401,7 @@ private static Map<String, MultiBidConfig> bidderToMultiBids(BidRequest bidReque
                 ? CollectionUtils.emptyIfNull(extRequestPrebid.getMultibid())
                 : Collections.emptyList();
 
-        final Map<String, MultiBidConfig> bidderToMultiBid = new HashMap<>();
+        final Map<String, MultiBidConfig> bidderToMultiBid = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
```

Comment:
Should we use apache `CaseInsensitiveMap` it is hash-based map, should be better for our case

Reply:
Also pls check similar occurences